### PR TITLE
Cookiecutter

### PR DESCRIPTION
**Pull request by cadenmyers13**

### CODE REVIEW

**Comment by cadenmyers13** [comment]
@sbillinge I had to rename `src/bg-mpl-stylesheets` to `src/bg_mpl_stylesheet` because black and flake8 were giving me errors in the pre-commit.

File: example/plot.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 plt.style.use(bg_mpl_style)
 
 # load PDF data
-r, gcalc, dr, dg, gdiff = loadData("./CdSe.fgr").T
+r, gcalc, dr, dg, gdiff = loadData("example/CdSe.fgr").T
```
Comment (sbillinge) [inline example/plot.py:11]: please check this?  I don't see this `CdSe.fgr`?

File: requirements/README.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# YOU MAY DELETE THIS FILE AFTER SETTING UP DEPENDENCIES!
```
Comment (sbillinge) [inline requirements/README.txt:1]: i think that this file needs to be deleted.

**Review by sbillinge — COMMENTED** [review]
couple of small things (please see inline) but this looks good when they are fixed.

File: example/plot.py (hunk 1 above)
Comment (cadenmyers13) [reply example/plot.py]: For some reason when I moved the files into `example` they did track. I had to do some renaming, duplicating, etc. to get them added. Should be good now.

**Pull request merged**